src/main/java/com/epam/emotionalhelp/controller/global/ErrorHandler.java

```diff
@@ -0,0 +1,62 @@
+package com.epam.emotionalhelp.controller.global;
+
+import com.epam.emotionalhelp.exception.NoAccessException;
+import com.epam.emotionalhelp.exception.NotAuthorizedException;
+import com.epam.emotionalhelp.exception.ResourceNotFoundException;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+import javax.validation.ConstraintViolationException;
+import java.rmi.AccessException;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+@RestControllerAdvice
+public class ErrorHandler {
+    @ResponseStatus(HttpStatus.NOT_FOUND)
+    @ExceptionHandler(ResourceNotFoundException.class)
+    public Map<String, String> resourceNotFound(ResourceNotFoundException e) {
+        String message = e.getMessage() == null
+                ? e.getClass().getSimpleName()
+                : e.getMessage();
+        return Map.of("code", "404",
+                "message", message);
```

Comment:
We have a response status code - I do not see a real value of specifying it in the response body.

Can you please remove this from map and keep only `@ResponseStatus` annotation?